### PR TITLE
added api-actrors

### PR DESCRIPTION
**Pull request by mariuszmmm**
Added API for actors (popularPeople)

### CODE REVIEW

**Review by Adrode — APPROVED** [review]
Very good job, nice and clean work!

File: src/api/fetchApi.js [hunk 1]
```diff
@@ -14,4 +14,16 @@ export const getMovieDetails = (id) => fetch(movieDetailsUrl + id + parametrs)
 
 export const getPopularMoviesApi = (page) => fetch(popularMoviesUrl + page)
   .then(response => response.json())
-  .catch(err => console.error(err));
\ No newline at end of file
+  .catch(err => console.error(err));
+
+export const getPopularPeopleApi = () => fetch(popularMoviesUrl)
```
Comment (Urbork) [inline src/api/fetchApi.js]: should this be `fetch(PopularPeopleUrl)` ?

File: src/api/fetchApi.js [hunk 2]
```diff
@@ -14,4 +14,16 @@ export const getMovieDetails = (id) => fetch(movieDetailsUrl + id + parametrs)
...
+  .then(response => response.json())
+  .catch(err => console.error(err));
+
+export const getPerson = () => fetch(popularMoviesUrl)
```
Comment (Urbork) [inline src/api/fetchApi.js]: like above, personUrl?

File: src/api/api.js [hunk 3]
```diff
@@ -18,6 +19,12 @@ export const genreUrl = `${url}genre/movie/list?api_key=${apiKey}`;
 export const movieDetailsUrl = `${url}movie/`;
 export const creditsUrl = `${url}movie/${movie_id}/credits?api_key=${apiKey}`;
 
+
+export const popularPeopleUrl = `${url}person/popular?api_key=${apiKey}`
+export const personUrs = `${url}person/${person_id}?api_key=${apiKey}`;
```
Comment (Urbork) [inline src/api/api.js]: personUrl instead of "personUrs"?

**Review by Urbork — COMMENTED** [review]
thanks to console logs i think popularPeople and person objects contain Movies data

**Comment by mariuszmmm** [comment]
changed api urls

**Comment by Urbork** [comment]
Thanks, nice work

**Pull request merged**